foreman/foreman.spec

```diff
@@ -621,6 +641,10 @@ if [ ! -e %{_datadir}/%{name}/config/initializers/encryption_key.rb -a \
   ln -s %{_sysconfdir}/%{name}/encryption_key.rb %{_datadir}/%{name}/config/initializers/
 fi
 
+# copy pregenerated apipie cache files
```

Comment:
Another thought, if we want Foreman core to always win, we could have plugins %exclude the conflicting files in their file list.